src/Streaker.java

```diff
@@ -47,4 +52,45 @@ public void handleVelocity(ArrayList<String> input) {
             this.addVelocity(0, Constants.CHARACTER_VELOCITY);
         }
     }
+
```

Comment:
Everything needs to be put onto correct level of rest of the codebase

Reply:
I'm not sure why this is still being flagged, it is a blank line separating two functions.